index.html

```diff
@@ -625,7 +625,13 @@ <h4 class="alert-heading">Updating Sonarr v2 to v3</h4>
                                     In macOS 10.12+ Gatekeeper App Translocation will prevent Sonarr from updating if it's being run directly from the download location. Therefore it MUST be moved to the Application folder.
                                 </div>
                             </dd>
-                            <dt><span class="step">3</span>Start Sonarr</dt>
+                            <dt><span class="step">3</span>Self-sign Sonarr</dt>
```

Comment:
This needs to be done in the relevant `.marko` file and then compiled with `yarn build` which will create `index.html`

Reply:
thanks for the tip. `macos.marko` is updated and `yarn build` executed